runtime/src/main/java/org/corfudb/util/NodeLocator.java

```diff
@@ -37,6 +37,10 @@
 
     /** The port number on the host the node is located on. */
     final int port;
+    
+    public String getConnectionString() {
```

Comment:
no javadoc. Also, why do we need this? .toString() gets the correct connection string, which may have a node id?